prow/servemux/servemux_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
I would like to keep these tests where they are to prove that we did not break hook

Reply:
Btw, this is a new file. No idea why GitHub is detecting a difference here :/